projects/essentials/src/ds/ui/interface_xml/interface_xml_importer.h

```diff
@@ -40,10 +40,10 @@ class XmlImporter {
 	// to the parent, saving you a sprite
 	static bool loadXMLto(ds::ui::Sprite* parent, const std::string& xmlFile, NamedSpriteMap& map,
 						  SpriteImporter customImporter = nullptr, const std::string& namePrefix = "",
-						  const bool mergeFirstSprite = false);
+						  const bool mergeFirstSprite = false, ds::cfg::Settings& override_map = ds::cfg::Settings(), ds::cfg::VariableMap local_map = ds::cfg::VariableMap());
 	static bool loadXMLto(ds::ui::Sprite* parent, XmlPreloadData& xmldata, NamedSpriteMap& map,
 						  SpriteImporter customImporter = nullptr, const std::string& namePrefix = "",
-						  const bool mergeFirstSprite = false);
```

Comment:
Updated the loadXMLto functions to take an override settings object and a variableMap object. The settings in the xml are updated by the override and the variableMap is updated by the settings in the xml and then used to do replacements in the xml.